edu.wpi.first.wpilib.plugins.cpp/src/main/java/edu/wpi/first/wpilib/plugins/cpp/wizards/newproject/WPIRobotCPPProjectCreator.java

```diff
@@ -85,6 +89,25 @@ public void finalize(IProject project) throws CoreException {
 	}
 	
 	private void updateVariables(IProject project) throws CoreException {
-		// TODO: implement C++ equivalent
+		try {
+			
+			IUserVarSupplier varSupplier = CCorePlugin.getDefault().getUserVarSupplier();
+			IStorableCdtVariables vars = varSupplier.getWorkspaceVariablesCopy();
+			
+			File dir = new File(WPILibCore.getDefault().getWPILibBaseDir() + File.separator + "user" + File.separator + "cpp" + File.separator + "lib");
+			File[] filesList = dir.listFiles();
+			String libNames = "";
+			for (File file : filesList) {
+				if (file.isFile()) {
+					libNames += "-l" + file.getName().substring(3).split("[.]")[0] + " ";
```

Comment:
Same lib comments as above. Any way that we can refactor this method? It seems line for line between these two classes.